packages/terra-switch/src/terra-dev-site/test/switch/LabelWrap.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import Switch from '../../../Switch';
+import styles from './SwitchCommonExample.module.scss';
+
+const cx = classNames.bind(styles);
+
+const SwitchWithLongLabelText = () => {
+  const [value, setValue] = useState(true);
+  return (
+    <div className={cx('container')}>
```

Comment:
Do we need to add this container or can we directly render the Switch? Adding extra styles around the container can be misleading IMO when looking at the raw test routes